src/service_name.json

```diff
@@ -44,6 +44,11 @@
     "AzureServiceName": "Azure CLI",
     "URL": ""
   },
+  {
+    "Command": "az cloud-service",
+    "AzureServiceName": "Azure Cloud Services",
```

Comment:
should it be `Azure Cloud Services (extended support)`?

Reply:
Right